src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/CarouselViewUITests.NoItemTemplate.cs

```diff
@@ -16,10 +16,9 @@ public CarouselViewNoItemTemplate(TestDevice device)
 		// Issue12777 (src\ControlGallery\src\Issues.Shared\Issue12777.cs
 		[Test]
 		[Category(UITestCategories.CarouselView)]
+		[FailsOnWindows]
```

Comment:
Should we change these attributes to require a description? Not that this is something that you're doing right now, but imo people should add a description as to why they're adding this.

Reply:
I've been thinking about it and if we are going to ignore tests for some reason, it makes sense to add a brief description indicating why.

Reply:
Applied changes, the ignore reason is now mandatory.